api-schema/xdb.yaml

```diff
@@ -157,6 +184,27 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/WorkerErrorResponse'
+  /api/v1/xdb/worker/process/rpc:
+    post:
+      summary: "invoking a Process RPC API"
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/ProcessRpcRequest'
```

Comment:
This name is hard to distinguish with the other two: `ProcessExecutionRpcRequest` and `ProcessExecutionRpcResponse`. 

How about `ProcessRpcWorkerRequest` and `ProcessRpcWorkerResponse`?